[sqlite3] Panic: runtime error

For some reason, the new version of Session panics with sqlite3 driver, although the example code has not changed.

## Steps to reproduce

1) Copy-n-paste the code below
2) `go run main.go` 2 times

## Example code

```go
package main

import (
	"github.com/gofiber/fiber"
	"github.com/gofiber/session"
	"github.com/gofiber/session/provider/sqlite3"
)

func main() {
	app := fiber.New()

	provider := sqlite3.New(sqlite3.Config{
	  DBPath:     "test.db",
	  TableName:  "session",
	})

	sessions := session.New(session.Config{
		Provider: provider,
	})

	app.Get("/", func(c *fiber.Ctx) {
		store := sessions.Get(c)
		defer store.Save()
		c.Send(store.ID())
	})

	app.Listen(3001)
}
```

## Error
```
2020/07/21 00:31:42 session error: session: sqlite3 index last_active already exists; provider: sqlite3;
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x40 pc=0x13721e5]

goroutine 1 [running]:
github.com/fasthttp/session/v2/providers/sqlite3.(*Provider).NeedGC(0x0, 0xc0000ce820)
        <autogenerated>:1 +0x5
github.com/fasthttp/session/v2.(*Session).SetProvider(0xc000582000, 0x1637700, 0x0, 0x1, 0x1)
        C:/Users/mysh/go/pkg/mod/github.com/fasthttp/session/v2@v2.2.0/session.go:64 +0x51
github.com/gofiber/session.New(0xc0004a7f10, 0x1, 0x1, 0x0)
        C:/Users/mysh/go/pkg/mod/github.com/gofiber/session@v1.2.4/session.go:148 +0x533
main.main()
        C:/Users/mysh/go/src/sqliteSessionsBug/main.go:17 +0x135
exit status 2
```

## Error (image)
![image](https://user-images.githubusercontent.com/44648612/87962431-c2a4ea80-cae9-11ea-8d8e-941b43de5f0c.png)

--- a/go.mod
+++ b/go.mod
@@ -3,8 +3,9 @@ module github.com/gofiber/session
 go 1.12
 
 require (
-	github.com/fasthttp/session/v2 v2.2.0
+	github.com/fasthttp/session/v2 v2.2.1
 	github.com/gofiber/fiber v1.13.1
 	github.com/gofiber/utils v0.0.9
+	github.com/klauspost/compress v1.10.10 // indirect
 	github.com/valyala/fasthttp v1.15.1
 )

